frontend/javascripts/oxalis/model/accessors/dataset_layer_transformation_accessor.ts

```diff
@@ -364,15 +381,15 @@ function isTranslationOnly(transformation?: AffineTransformation) {
   return scale.equals(NON_SCALED_VECTOR) && quaternion.equals(IDENTITY_QUATERNION);
 }
 
-function isRotationOnly(transformation?: AffineTransformation) {
+function isRotationAndMirrorMaybeOnly(transformation?: AffineTransformation) {
   if (!transformation) {
     return false;
   }
   const threeMatrix = new THREE.Matrix4()
     .fromArray(nestedToFlatMatrix(transformation.matrix))
     .transpose();
   threeMatrix.decompose(translation, quaternion, scale);
-  return translation.length() === 0 && scale.equals(NON_SCALED_VECTOR);
+  return translation.length() === 0 && scale.length() === NON_SCALED_VECTOR.length();
```

Comment:
hmm, similarily to the AI, I'm also wondering whether this is the best approach. I think, I would have preferred to take Math.abs of the vector and then check that it equals 1,1,1.
currently, something like (sqrt(3),0,0) could result in the same length as (1,1,1), couldn't it? would be improbable, but still not so clean.